bindings/cs/rl.net/LiveModel.cs

```diff
@@ -229,6 +229,36 @@ public static int LiveModelReportActionTaken(IntPtr liveModel, IntPtr eventId, I
                 return LiveModelReportActionTakenNative(liveModel, eventId, apiStatus);
             }
 
+            [DllImport("rlnetnative", EntryPoint = "LiveModelReportEpisodicActionTaken")]
+            private static extern int LiveModelReportEpisodicActionTakenNative(IntPtr liveModel, IntPtr episodeId, IntPtr eventId, IntPtr apiStatus);
```

Comment:
Not sure why do we need to expose notion of episodic in feedback methods.
We have generic notion of primary and secondary id which are also used in ccb and slates. Right now there is no per-slot activation for them but it could be in theory and there is no difference in implementation on client side.
So maybe either reuse ccb methods completely or just introduce custom wrappers on very last c# level to adjust signature (eventId/slotId -> episodeId/eventId)?

Reply:
This is to abstract the feedback from the real implementation. You are right that "it could be in theory and there is no difference in implementation on the client side.". So the underlying implementation at C++ layer could be the same.

This is a question of at what layer we want to expose the internal implementation. Since all C# codes are easy to be decompiled, I think matching the entry of the c++ method with the c# wrapper is better.

On the other hand, reusing CCB methods on the Episodic + CB implementation is really confusing to me.

Reply:
Fixed.